python/pyarrow/tests/test_cython.py

```diff
@@ -136,9 +138,9 @@ def test_cython_api(tmpdir):
             delim, var = ':', 'LD_LIBRARY_PATH'
 
         subprocess_env[var] = delim.join(
-            pa.get_library_dirs() + [subprocess_env.get(var, '')]
+            pa.get_library_dirs() +
+            [subprocess_env.get(var, '')] + [str(tmpdir)]
```

Comment:
I'm curious, why was it necessary to add `tmpdir` here? Ideally, `pa.get_library_dirs()` is sufficient.

Reply:
This is a leftover from debugging a Win CI failure. Will remove it 👍